tools/precommit.py

```diff
@@ -170,6 +202,18 @@ def get_os_dependency_exclude_module(exclude_module):
                               '--compile-flag=--sysroot=' + tizen_root
                               ] + os_dependency_module['linux'] + build_args)
 
+    elif test == "artik053":
+        for buildtype in option.buildtype:
+            tizenrt_root = fs.join(path.PROJECT_ROOT, 'deps', 'tizenrt')
+            setup_tizenrt_root(tizenrt_root)
+            build(buildtype, ['--target-arch=arm',
+                              '--target-os=tizenrt',
+                              '--target-board=artik05x',
+                              '--sysroot=' + tizenrt_root + '/os',
+                              '--jerry-heaplimit=128',
+                              '--clean',
+                              ] + os_dependency_module['tizenrt'] + build_args)
```

Comment:
In my opinion it is not sufficient. Please compile static image of Tizen RT. You just compile IoT.js static library but you doesn’t compile any applications using it so missing symbols are not detected.